mambaSharedFramework/HLSValueTypes.swift

```diff
@@ -132,29 +132,18 @@ public func ==(lhs: HLSEncryptionMethodType, rhs: HLSEncryptionMethodType) -> Bo
 }
 
 /// Represents a minimum required HDCP level needed to play content.
-public struct HLSHDCPLevel: Equatable, FailableStringLiteralConvertible {
-    public let type: HDCPLevel
-    public enum HDCPLevel: String {
-        /// Indicates that the content does not require output copy protections.
-        case none = "NONE"
-        /// Indicates that the Variant Stream could fail to play unless the output is protected by High-bandwidth
-        /// Digital Content Protection (HDCP) Type 0 or equivalent.
-        case type0 = "TYPE-0"
-        /// Indicates that the Variant Stream could fail to play unless the output is protected by HDCP Type 1 or
-        /// equivalent.
-        case type1 = "TYPE-1"
-    }
+public enum HLSHDCPLevel: String, Equatable, FailableStringLiteralConvertible {
```

Comment:
I think this feels better like- more modern swift usage.

I'm ok with breaking from some of the existing patterns used in the project if we can bring it up to speed with mode current swift conventions.